src/form/FormikHelpers.tsx

```diff
@@ -171,29 +165,11 @@ export const FormikSubmitButton = ({
   label = "Save",
   inProgressLabel = "Saving...",
   isCreate,
-  allowNavigation,
 }: FormikSubmitButtonProps) => {
   const formik = useFormikContext();
-  const history = useHistory();
-
   let saveButtonTooltip: TooltipProps["title"] = "";
   let saveEnabled = true;
 
-  // On create, the save button is initially enabled so the user can click it
-  // to see the full set of validation errors.
-  // On edit, it should only be enabled once they've made changes.
-  useEffect(() => {
```

Comment:
If you hit "cancel" on this, it blocks history when you try to shift routes via the sidebar, then "unblocking" doesn't catch any new navigation. it just simply does not work right now